test/lint/lint-rpc-help.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2018 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+export LC_ALL=C
+
+EXIT_CODE=0
+non_autogenerated_help=$(grep --perl-regexp --null-data --only-matching 'runtime_error\(\n\s*".*\\n"\n' $(git ls-files -- "*.cpp"))
```

Comment:
I'm afraid BSD grep does not support `--perl-regexp` and `--null-data` :-\

Reply:
Perhaps the linter can just `exit 0` with an information message if BSD grep is used?